doc/plugins.rst

```diff
@@ -5,73 +5,89 @@ tox plugins
 
 .. versionadded:: 2.0
 
-With tox-2.0 a few aspects of tox running can be experimentally modified
-by writing hook functions.  The list of of available hook function is
-to grow over time on a per-need basis.
+A growing number of `pluggy`_ hooks make tox extendable by writing plugins.
 
 
-writing a setuptools entrypoints plugin
+Writing a setuptools entrypoints plugin
 ---------------------------------------
 
-If you have a ``tox_MYPLUGIN.py`` module you could use the following
-rough ``setup.py`` to make it into a package which you can upload to the
-Python packaging index:
+You can create a new tox plugin with all bells and whistles via `Cookiecutter`_ (see `cookiecutter-tox-plugin <https://github.com/tox-dev/cookiecutter-tox-plugin>`_):
 
-.. code-block:: python
-
-    # content of setup.py
-    from setuptools import setup
+.. code-block:: console
 
-    if __name__ == "__main__":
-        setup(
-            name='tox-MYPLUGIN',
-            description='tox plugin decsription',
-            license="MIT license",
-            version='0.1',
-            py_modules=['tox_MYPLUGIN'],
-            entry_points={'tox': ['MYPLUGIN = tox_MYPLUGIN']},
-            install_requires=['tox>=2.0'],
-        )
+    $ pip install -U cookiecutter
+    $ cookiecutter gh:tox-dev/cookiecutter-tox-plugin
 
-If installed, the ``entry_points`` part will make tox see and integrate
-your plugin during startup.
+This will create a complete pypi-releasable, documented project with license, documentation,
+CI builds and other bells and whistles.
```

Comment:
> documented project with license, documentation,

Is a little redundant. "bells and whistles" has also been used recently Maybe...

> This will create a complete, PyPI-ready package with a license, documentation, tests, and CI via Travis and Appveyor.

Reply:
Definitely too many bells and whistles :)